packages/engine/lib/error/Cargo.toml

```diff
@@ -8,8 +8,10 @@ provider = { path = "../provider" }
 
 tracing-error = { version = "0.2.0", optional = true }
 
+[build-dependencies]
+rustc_version = "0.2.3"
```

Comment:
Is this implied by the toolchain file?

Reply:
This is a thin wrapper around `rustc -vV` used in the build script to get the release channel.